README.asciidoc

```diff
@@ -393,16 +392,16 @@ selection's cursor. Some additional keys are recognised in insert mode:
 
  * `<c-r>`: insert contents of the register given by next key
  * `<c-v>`: insert next keystroke directly into the buffer,
-    without interpreting it.
+    without interpreting it
 
- * `<c-u>`: commit changes up to now as a single undo group.
+ * `<c-u>`: commit changes up to now as a single undo group
 
  * `<a-;>`: escape to normal mode for a single command
 
 Movement
 ~~~~~~~~
 
-See <<Appending>> below for instructions on extending (appending to) the current selection, in order to select more text in multiple steps.
+See <<Appending>> below for instructions on extending (appending to) the current selection in order to select more text.
```

Comment:
"in multiple steps" could also be "incrementally" but it also seems fine to drop it

Reply:
Unless someone wants it otherwise, I'm inclined to dropping it.